force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -288,27 +305,39 @@ public with sharing class FlowControlLoopsCollections {
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+        Decimal sum = 0;
+        for (Decimal cash : moneyInWallet) {
+            sum += cash;
+         if (sum > 40) {
+              break;
+         }
+        }
+        return sum; 
     }
 
+
     /**
      * Question 18
      * Create a map that has five key-value pairs to the map, where the keys are the names of fruits and the values are their quantities.
      * Example: addItemsToMap() should return "Apples => 5, Oranges => 10, Bananas => 15, Pears => 20, Grapes => 25"
      * @return The Map of fruits and their quantities.
      */
     public static Map<String, Integer> addItemsToMap() {
-        // Initialize an empty Map
-
+    Map<String, Integer> Fruit = new Map<String, Integer>();// Initialize an empty Map
+        
+        Fruit.put('Apples', 5);
+        Fruit.put('Oranges', 10);
+        Fruit.put('Bananas', 15);
+        Fruit.put('Pears', 20);
+        Fruit.put('Grapes', 25);
         // Add key-value pairs to the Map
         // add Apples => 5
         // add Oranges => 10
         // add Bananas => 15
         // add Pears => 20
         // add Grapes => 25
 
-        return null; // Replace null with the variable you used to store the result
+        return Fruit; // Replace null with the variable you used to store the result
```

Comment:
Good work. I like to use the Map constructor for these if I know my values when I am writing the code. 

```
 public static Map<String, Integer> addItemsToMap() {

        return new Map<String, Integer>{
                'Apples' => 5,
                'Oranges' => 10,
                'Bananas' => 15,
                'Pears' => 20,
                'Grapes' => 25
        };
    }
```